source/includes/fact-system-vars.rst

```diff
@@ -26,25 +28,144 @@
        - ``2``: For internal use only.
      - ``2``
      - ``0``
+
    * - ``max_nested_table_depth``
      - integer
      - Specifies the maximum number of unique nested field paths that
        ``mongosqld`` maps to a relational table for a collection.
      - ``50``
      - ``50``
+
    * - ``max_num_columns_per_table``
      - integer
      - The maximum number of unique fields that ``mongosqld`` maps to
        relational columns for a collection.
      - ``1000``
      - ``1000``
+
+   * - ``metrics_backend``
+     - string
+     - Determines where |bi-short| stores query metrics.
+ 
+       Valid values are:
+
+       - ``off``: No metrics are stored.
+
+       - ``log``: Query metrics are written to the ``mongosqld``
+         :doc:`log </reference/log-messages/>`.
+
+       - ``stitch``: Query metrics are written to a `Stitch project
+         <https://docs.mongodb.com/stitch/>`__.
+
+     - ``off``
+     - ``off``
+
+   * - ``mongodb_max_connection_size``
+     - integer
+     - The maximum size of memory in bytes that may be allocated for
+       evaluating any query on any given client connection. A value of
+       ``0`` specifies no limit.
+     - ``0``
+     - ``0``
+
+   * - ``mongodb_max_server_size``
+     - integer
+     - The maximum size of memory in bytes that may be allocated for
+       evaluating any query on |bi-short|. A value of ``0`` specifies
```

Comment:
Perhaps "all queries" would be clearer than "any query" here?

Reply:
Done.